### ISSUE TITLE
support to Template tag

### ISSUE THREAD

**Issue by dashaus** [Microsoft/TypeScript#5199]
Hi,
There is support for [Template](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template) tag?

```typescript
let templ = <HTMLTemplateElement>document.querySelector("#template");
let elem = templ.content.querySelector("#container");
```

**Comment by DanielRosenwasser** [comment]
It doesn't appear so; @zhengbli might know more.

**Comment by mhegazy** [comment]
@zhengbli is this ready to be ported?

**Comment by zhengbli** [comment]
@mhegazy yes I'll update today.

**Comment by mhegazy** [comment]
thanks!

**Comment by panuhorsmalahti** [comment]
Is HTMLTemplateElement supported in 1.7.5, or is the support coming to 1.8.0? For some reason I don't see the HTMLTemplateElement in 1.7.5. The file dom.generated.d.ts is not in the release, and HTMLTemplateelement is not found in lib.dom.d.ts.

**Comment by zhengbli** [comment]
@panuhorsmalahti it should be released with 1.8, in the lib.dom.d.ts and lib.d.ts
